### PR TITLE
[Storage] Fix the issues in Az.Storage Preview module.

### PR DESCRIPTION
**Pull request by blueww**
#### Fixed issues
1. Use the released "microsoft.azure.management.storage" SDK, by remove the private signed "microsoft.azure.management.storage" SDK. The released the SDK reference JSON  10, can resolve the JSON 9 issue before
2. Fix the New-AzureStorageContext cmdlet not found issue. Should be caused by the cmdlets name in code is in old format.

Has tested the build out Az.Storage module with Az.Profile module in \\aaptfile01\adxsdk\PowerShell\2018_11_19_AzureRM.Storage\pkgs, and it works well in the above 2 issues.

#### Remain Issue
The only remain issue is: some cmdlets old name not work, but new name work, like `Enable/Disable-AzStorageStaticWebsite` works, but `Enable/Disable-AzureStorageStaticWebsite` not work. It seems like cmdlet alias issue. I have added the impact cmdlets to `src\ResourceManager\Profile\Commands.Profile\AzureRmAlias\Mappings.json`, but seems the issue still exist.

> Enable-AzureStorageStaticWebsite : The term 'Enable-AzureStorageStaticWebsite' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again. The impacted cmdlets are:

- Enable/Disable-AzureStorageStaticWebsite
- Invoke-AzureRmStorageAccountFailover
- Get/Set/Remove-AzureRmStorageAccountManagementPolicy

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.json [hunk 1]
```diff
@@ -2281,6 +2281,9 @@
     "Get-AzStorageAccountNetworkRuleSet": "Get-AzureRmStorageAccountNetworkRuleSet",
     "Add-AzStorageAccountNetworkRule": "Add-AzureRmStorageAccountNetworkRule",
     "Remove-AzStorageAccountNetworkRule": "Remove-AzureRmStorageAccountNetworkRule",
+    "Set-AzStorageAccountManagementPolicy": "Set-AzureRmStorageAccountManagementPolicy",
```
Comment (markcowl) [inline src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.json]: There is no need for this.  These cmdlet aliases will exist by default.

Comment (blueww) [reply src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/Mappings.json]: the change to mappings.json is reverted.

**Comment by blueww** [comment]
@markcowl 
I have fixed you comments ,would you please help to look?

**Comment by blueww** [comment]
@markcowl 

I have fixed the comments from you. Would you please help to review and merge the PR if not other comments.

**Pull request merged**